data/human/human missions.txt

```diff
@@ -6832,6 +6832,90 @@ mission "FW Refugees to Humanika"
 		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
 		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`
 
+mission "Derelict Treasure Map: Ion Cannons"
+	landing
+	name "Derelict Treasure Map"
+	description "Follow the treasure map you found in a derelict ship to <destination>."
+	destination "New Argentina"
+	to offer
+		has "derelict treasure map: ion cannons"
+	on offer
+	on offer
+		conversation
+			`Upon landing on <origin> you take out the treasure map you recovered from the derelict ship. The map gives detailed instructions about where the treasure was buried on the sparsely populated world of <destination>. However, the map's creator claims to have obtained some kind of alien weapons that disrupts a ship's electrical systems. The outlandish claim seems to bring doubt to the validity of the map.`
+			choice
+				`	(Follow the map.)`
+					accept
+				`	(Throw the map away, it's probably fake.)`
+					decline
+	on complete
+		conversation
+			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shoveling the hard soil you're met with a thunk as you strike metal.`
+			branch hai
+				has "First Contact: Hai: offered"
+			`	Opening the cache reveals a large store of credits along with the alien weapons the map's owner wrote about. You marvel at their strange design and wonder at their power.`
+			choice
+				`	(Loot the treasure.)`
+					goto end
+			label hai
+			`	Opening the cache reveals a large store of credits along with two Hai Ion Cannons. These must be the alien weapons the map's owner wrote about.`
+			choice
+				`	(Loot the treasure.)`
+					goto end
+			label end
+			action
+				payment 386000
+				outfit "Ion Cannon" 2
+			`	You grab the <payment> credits, and with the assistance of some machinery you bring the weapons back to your ship.`
+
+mission "Derelict Family Investment 1"
+	name "Lost Family Investment"
+	description "Return the credits you found in a derelict ship to the family it belongs to."
+	landing
+	to offer
+		has "derelict family investment"
+	destination "Sundive"
+	on offer
```

Comment:
I don't think the first payment replacement in this conversation is going to work because there is no on offer or on complete that contains a payment. The second payment replacement will work because the action containing the payment is directly above it. Therefore, we could use an apparent payment here to fix the first replacement.
```suggestion
	"apparent payment" 325000
	on offer
```